### PR TITLE
chore(docs): tweak backers/donors

### PR DESCRIPTION
**Pull request by tmorehouse**
### Describe the PR

Adjusts the algorithm for processing the backers section, to make sure backers who started before the tiers were set up are shown.

Also only links to donors website once a certain donation threshold is reached, to prevent questionable sites from getting cheap backlinks to their webpage.

### PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Enhancement
- [ ] ARIA accessibility
- [x] Documentation update
- [x] Other (please describe)

**Does this PR introduce a breaking change?** (check one)

- [x] No
- [ ] Yes (please describe)

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (i.e. `[...] (fixes #xxx[,#xxx])`, where "xxx" is the issue number)
- [x] It should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] The title should follow the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. `fix(alert): not alerting during SSR render`, `docs(badge): update pill examples`, `chore(docs): fix typo in README`, etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [ ] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] Includes any needed TypeScript declaration file updates
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [ ] CodeCov for patch has met target
- [x] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (Does it affect screen reader users or keyboard only users? Clickable items should be in the tab index, etc.)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: docs/components/contributors.vue [hunk 1]
```diff
@@ -228,15 +261,19 @@ export default {
     processBackers(backers = []) {
       // Backers are provided in reverse chronological order
       // so we sort by larger amount first, then by date
-      // Limit to top 16 backers
-      this.backers = backers.sort(this.sortCompare).slice(0, MAX_BACKERS)
+      // Limit to top N backers
+      // this.backers = backers.sort(this.sortCompare).slice(0, MAX_BACKERS)
```
Comment (jackmu95) [inline docs/components/contributors.vue]: @tmorehouse Can we get rid of this commented out code?

File: docs/components/contributors.vue [hunk 2]
```diff
@@ -260,7 +297,9 @@ export default {
       // Bronze sponors are people/organizations with a recurring (active) bronze sponorship
       this.makeOcRequest(this.processBronzeSponsors.bind(this), { tierSlug: 'bronze-sponsors' })
       // Backers are people/organizations with recurring (active) donations
-      this.makeOcRequest(this.processBackers.bind(this), { tierSlug: 'backers' })
+      // Some backers are not tagged as "backers" slug (null slug), but have status "active"
+      // this.makeOcRequest(this.processBackers.bind(this), { tierSlug: 'backers' })
```
Comment (jackmu95) [inline docs/components/contributors.vue]: @tmorehouse Same here?

File: docs/components/contributors.vue (hunk 1 above)
Comment (tmorehouse) [reply docs/components/contributors.vue]: Yeah, I just had it in there in case the following code didn't work ;-)

Comment (tmorehouse) [reply docs/components/contributors.vue]: fixed

File: docs/components/contributors.vue (hunk 2 above)
Comment (tmorehouse) [reply docs/components/contributors.vue]: fixed

**Pull request merged**